app/controllers/api/v1/events_controller.rb

```diff
@@ -1,5 +1,40 @@
-class Api::V1::EventsController < ApplicationController
+class Api::V1::EventsController < ApiController
   def show
     @event = Event.find(params[:id])
   end
+
+  def create
+    @event = Event.new(event_params)
+
+    if @event.save
+      render
```

Comment:
So, when we create something in a RESTful we'll want to send back a meaningful response code.

This `render` will do `:ok` or `200` ... which is nice, it succeeded.

But,

`render status: :created`
 is nicer because it'll tell the client that yes, something really was created.

This is a great resource for knowing what codes to send

https://github.com/interagent/http-api-design#return-appropriate-status-codes